src/helpers/customHooks/useWeather.js

```diff
@@ -78,7 +88,23 @@ const useWeather = () => {
           }
         );
     }
-  }, [city]);
+  }, [debouncedSearchTerm]);
+
+  function fetchWeatherUsingCoordinates({ lat, lng }) {
```

Comment:
As suggested by Priya, we are trying to use arrow functions instead of class or function keywords, to keep a cleaner and more uniform code. It will be great if you could change this function and the function LocationMarker in src/components/MyMap/index.js to an arrow function.

Reply:
Yes...I'll make those changes to arrow functions now

Reply:
@sumana2001 I've pushed the changes changing to arrow functions